core/templates/dev/head/pages/collection-player-page/collection-player-page.directive.html

```diff
@@ -47,12 +47,25 @@ <h2 ng-if="$ctrl.collectionPlaythrough.hasFinishedCollection()" class="oppia-pag
       <a ng-repeat="node in $ctrl.collection.getCollectionNodes()"
          ng-href="<[$ctrl.getExplorationUrl(node.getExplorationId())]>"
          ng-style="{position: 'absolute', left: '<[$ctrl.pathIconParameters[$index].left]>', top: '<[$ctrl.pathIconParameters[$index].top]>'}">
+
+        <div ng-style="{width: '160px',
+              position: 'absolute',
+              color: '#006553',
+              'font-family' : 'Capriola, Roboto, Arial, sans-serif',
+              'font-size': '18px',
+              bottom: '70%',
+              left: '<[$ctrl.getExplorationTitleStylePosition($index)]>'
+                }">
+          {{node._explorationSummaryObject.title}}
+        </div>
+
         <svg class="protractor-test-collection-exploration"
              width="100"
              height="125"
              xmlns="http://www.w3.org/2000/svg"
              xmlns:xlink="http://www.w3.org/1999/xlink"
-             version="1.1">
+             version="1.1"
+             ng-style="{position:'relative',top:'20px'}">
```

Comment:
since these attributes are static, why can't we use the style tag directly.

Reply:
I thought of letting angular take care of the styling, anyways yup it's not necessary. I'll put that up in the style tag.